x/auth/types/account.go

```diff
@@ -298,6 +298,32 @@ func (ma *ModuleAccount) UnmarshalJSON(bz []byte) error {
 	return nil
 }
 
+// Bech32toValidatorAddresses returns []ValAddress from a list of Bech32 string addresses.
+func Bech32toValidatorAddresses(validators []string) ([]sdk.ValAddress, error) {
```

Comment:
```suggestion
func Bech32toValAddresses(validators []string) ([]sdk.ValAddress, error) {
```

Reply:
Also, `types/address.go` might be a better place for these.

Reply:
do you mean `types/address.go` in `auth` or `types` package ?

Reply:
types package. sorry for the confusion